identity-server/migrations/IdentityServerDb/IdentityServerDb.csproj

```diff
@@ -31,17 +31,20 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\hosts\Config\Clients.cs">
-      <Link>Configuration\Clients.cs</Link>
+    <Compile Include="..\..\hosts\Config\TestClients.cs">
```

Comment:
Might as well do the wildcard thing here too, at least for the Configuration subdir.